spec/features/blacklight_customizations/hierarchical_date_facet_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'Hierarchical date facet' do
+  before do
+    visit root_path
+    click_link_or_button 'search'
+  end
+
+  it 'includes an expandable date facet' do
+    expect(page).to have_content('1 - 18 of 18')
```

Comment:
In a recent PR of mine (https://github.com/sul-dlss/earthworks/pull/1337) we decided that hardcoding fixture counts into the tests was going to cause annoyance in the future.
This is more targeted because it's counting a field that will probably not be updated frequently. But if there was a way to test this without using counts, I'd go for that. If not, I think it's fine.

Reply:
ah yeah, good point, will think for a bit about trying to get out of using counts...

Reply:
ok, so i wanted to keep somewhat precise search behavior expectations, to confirm that faceting wasn't happening when it shouldn't but also that faceting narrowed results appropriately when applied.  but the worry about test fragility and annoyance totally makes sense.

so given that the main worry in the other PR's discussion seemed to be about frequent annoyance when adding unrelated fixtures, and since it seemed less common that we'd regularly add or remove fixtures in the 1990-1999 date range, i tried to find a middle ground: i kept the hardcoded count for the post-facet results; and for the prefacet expectations, i just looked for the initial total result count, and made sure that stayed unchanged when expanding (but not choosing) a facet in the tree.

hopefully that's ok, but happy to revisit if that still feels too brittle!